tests/conftest.py

```diff
@@ -686,3 +719,19 @@ def default_model_storage(tmp_path: Path) -> ModelStorage:
 @pytest.fixture()
 def default_execution_context() -> ExecutionContext:
     return ExecutionContext(GraphSchema({}), uuid.uuid4().hex)
+
+
+@pytest.fixture(autouse=True)
+def use_temp_dir_for_cache(
```

Comment:
Is this used?

Reply:
yes, it's crucial! Set the cache to a temp dir for every training we do

Reply:
oh missed the `autouse`!